pkg/client/clientset/versioned/clientset.go

```diff
@@ -31,6 +32,9 @@ type Interface interface {
 	CephV1alpha1() cephv1alpha1.CephV1alpha1Interface
 	// Deprecated: please explicitly pick a version if possible.
 	Ceph() cephv1alpha1.CephV1alpha1Interface
+	CockroachdbV1alpha1() cockroachdbv1alpha1.CockroachdbV1alpha1Interface
+	// Deprecated: please explicitly pick a version if possible.
+	Cockroachdb() cockroachdbv1alpha1.CockroachdbV1alpha1Interface
```

Comment:
Why are you including a deprecated item in the interface? Can't you just omit this and start with ones that are versioned?

Reply:
This is all generated code from `k8s.io/code-generator`, this is the pattern that they automatically generate for you.